tests/converter/converter_test.py

```diff
@@ -93,6 +93,7 @@ def run_test(test_name, c4q_exe, h5diff_exe, conv_inp, gold_file, expect_fail, e
                       print("  pass")
                       return True
                    else:
+                      print("h5diff reported a difference")
```

Comment:
But this does report now there is a difference and it can be then looked at with -v?

Reply:
This points out the failure is due to h5diff (vs. the current situation where the output is simply "FAIL")